Services/Calendar/templates/default/tpl.week_view.html

```diff
@@ -52,9 +69,37 @@
 			</td>
 			<!-- END time_txt -->
 			<!-- BEGIN not_full_day_event -->
-			{CONTENT_EVENT}
+				<!-- BEGIN scrd_day_cell -->
+				<td id="show_{DAY_CELL_NUM}" class="{TD_CLASS}" style="{TD_STYLE}" valign="top" rowspan="{TD_ROWSPAN}"
+					onmouseover="showNewAppLink({DAY_ID})">
+					<!-- BEGIN scrd_not_empty -->
+					<div class="calevent" style="{DIV_STYLE}">
+						{GRID_CONTENT}
+					</div>
+					<!-- END scrd_not_empty -->
+				</td>
+				<!-- END scrd_day_cell -->
+				<!-- BEGIN day_cell -->
+				<td id="show_{DAY_CELL_NUM}" class="{TD_CLASS}" style="{TD_STYLE}" valign="top" rowspan="{TD_ROWSPAN}"
+					onmouseover="showNewAppLink({DAY_ID})" onmouseout="hideNewAppLink()">
+					<!-- BEGIN new_app_link -->
+					<div id="new_link_{DAY_NEW_ID}" style="float: right;visibility:hidden">
+						{DAY_NEW_APP_LINK}
+					</div>
+					<!-- END new_app_link -->
+					<!-- BEGIN not_empty -->
+					{GRID_CONTENT}
+					<!-- END not_empty -->
+				</td>
+				<!-- END day_cell -->
 			<!-- END not_full_day_event -->
 		</tr>
 		<!-- END time_row -->
 	</tbody>
-</table>
\ No newline at end of file
+</table>
+<script>
+    $('.calfullcontent').each(function(){
+        var fs = parseInt($(this).parent().height());
+        $(this).css('height',fs);
+    });
```

Comment:
This would do the trick:
```
	$('.calfullcontent').each(function(){
		var fs = parseInt($(this).parents(".calevent").height());
		$(this).css('height',fs);
	});
```

However, why is this inline. Please only do not mix js into templates!